bin/MorphingMSSMvsSM.cpp

```diff
@@ -626,88 +768,106 @@ int main(int argc, char **argv) {
   std::vector<int> em_control_category = {2}; // Control region for em channel
   std::vector<int> mssm_btag_categories = {35,36,37}; // b-tagged MSSM-like categories with mt_tot as discriminator
   std::vector<int> mssm_nobtag_categories = {32,33,34}; // non-btagged MSSM-like categories with mt_tot as discriminator
-  std::vector<int> sm_signal_category = {1}; // category for the SM signal
+  std::vector<int> sm_signal_category = {1,3,4,5,6,7}; // category for the SM signal
```

Comment:
Maybe we could have put the split categories to some different numbers like e.g. above 100 or so. I think it is not necessary to change it now but would be a little more comprehensive if the control region was not in between in the numbering.

Reply:
yes agreed. In principle we could do so. We just would need to be consistent in the systematic file with this too. I can do that in a separate, direct commit (or during my updates for model-dependent analysis with low-mass signals)